### PR TITLE
add Alibaba Cloud Provider docs

### PR DESCRIPTION
**Pull request by haoshuwei**
fix issue [#1516 ](https://github.com/spinnaker/spinnaker.github.io/issues/1516)
Signed-off-by: haoshuwei <haoshuwei24@gmail.com>

### CODE REVIEW

File: setup/install/providers/kubernetes-v2/ack/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+# Create a cluster
+
+You can create a kubernetes cluster on ACK using [console](https://cs.console.aliyun.com) 
+as shown in the official documentation. You can also use [Alibaba Cloud CLI](https://github.com/aliyun/aliyun-cli) 
+or [Terraform ](https://www.terraform.io/docs/providers/alicloud/r/cs_kubernetes.html) tools to automate provisioning your clusters.
+
+# Download kubectl configuration file on hal node
+
+Run the follwing command on the `hal` node:
```
Comment (duftler) [inline setup/install/providers/kubernetes-v2/ack/index.md]: s/follwing/following

File: setup/install/providers/kubernetes-v2/ack/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+hal config provider kubernetes account add my-k8s-v2-acct \
+    --provider-version v2 \
+    --context $(kubectl config current-context)
+```
+Enable the Kubernetes provider and artifacts feature:
+
+```bash
+hal config provider kubernetes enable
+hal config features edit --artifacts true
+```
```
Comment (duftler) [inline setup/install/providers/kubernetes-v2/ack/index.md]: Could use a 'Next Steps' section like these other providers have:

https://github.com/spinnaker/spinnaker.github.io/blame/master/setup/install/providers/kubernetes-v2/gke/index.md#L60-L63
https://github.com/spinnaker/spinnaker.github.io/blame/master/setup/install/providers/kubernetes-v2/aws-eks.md#L169-L171

**Comment by haoshuwei** [comment]
@duftler Thanks for your comments, I have updated the index file.

**Review by ezimanyi — APPROVED** [review]

**Review by duftler — APPROVED** [review]

File: setup/install/providers/kubernetes-v2/ack/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+{% include toc %}
+
+This page describes how to set up a Kubernetes cluster on
+[ACK](https://www.alibabacloud.com/product/kubernetes) to be used as a Spinnaker
+Kubernetes v2 provider.
+
+# Create a cluster
+
+You can create a kubernetes cluster on ACK using [console](https://cs.console.aliyun.com)
```
Comment (dorbin) [inline setup/install/providers/kubernetes-v2/ack/index.md]:
```suggestion
You can create a Kubernetes cluster on ACK using [console](https://cs.console.aliyun.com) 
```

File: setup/install/providers/kubernetes-v2/ack/index.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+{% include toc %}
+
+This page describes how to set up a Kubernetes cluster on
+[ACK](https://www.alibabacloud.com/product/kubernetes) to be used as a Spinnaker
+Kubernetes v2 provider.
+
+# Create a cluster
+
+You can create a kubernetes cluster on ACK using [console](https://cs.console.aliyun.com) 
+as shown in the official documentation. You can also use [Alibaba Cloud CLI](https://github.com/aliyun/aliyun-cli)
```
Comment (dorbin) [inline setup/install/providers/kubernetes-v2/ack/index.md]: @haoshuwei can you add a link to the documentation?

**Review by dorbin — COMMENTED** [review]
Two more minor comments, @haoshuwei

**Comment by haoshuwei** [comment]
@dorbin Thank you for your comments and i'm sorry for the late update. I have add a document link to help users create a Kubernetes cluster using ACK console.

**Review by dorbin — APPROVED** [review]

**Pull request merged**